src/background/Wallet/Wallet.ts

```diff
@@ -1316,7 +1325,18 @@ export class Wallet {
     );
 
     const keypair = this.getKeypairByAddress(currentAddress);
-    return SolanaSigning.signTransaction(transaction, keypair);
+    const result = SolanaSigning.signTransaction(transaction, keypair);
+    const { mode } = await this.assertNetworkMode({
+      id: createChain('solana'),
+    }); // MUST assert even if result is not used
```

Comment:
What does this comment mean?

Reply:
It's copied, it's simply a reminder 🤷‍♀️ E.g. one may question why we're asserting correct mode if solana doesn't exist in testnet mode, and this is a reinforcer that it should still be done :)